feature-libs/cart/saved-cart/styles/_add-to-saved-cart.scss

```diff
@@ -8,14 +8,13 @@ cx-add-to-saved-cart {
 
     .cx-action-link {
       &:nth-child(n + 2) {
-        margin-inline-end: 0;
+        position: relative;
```

Comment:
Changes to styling are also considered breaking changes. But fortunately we have a way to handle these too with similar feature flags to DOM breaking changes:
https://sap.github.io/spartacus-docs/css-architecture/#style-versioning

We should leave clean up instructions in a similar way to the discussion here:
https://github.com/SAP/spartacus/pull/18268#discussion_r1432623453

@developpeurweb Do you have any other comments in this regard?

Reply:
@Zeyber no further comments, all good.